pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1020,12 +1020,16 @@ def _refresh_slaves(self, dev_cfg):
 
     def _refresh_parent_vlanid(self, dev_cfg):
         if dev_cfg.get_device_type() == NM.DeviceType.VLAN:
+            vlanidstr = parent = ""
```

Comment:
Please rename this variable. Sounds to me like someones name :)

Reply:
hehe, I'll try